R/helper_exec.R

```diff
@@ -23,6 +23,9 @@ future_map = function(n, FUN, ..., MoreArgs = list()) {
   if (getOption("mlr3.debug", FALSE)) {
     lg$info("Running experiments sequentially in debug mode with %i iterations", n)
     mapply(FUN, ..., MoreArgs = MoreArgs, SIMPLIFY = FALSE, USE.NAMES = FALSE)
+  } else if (requireNamespace("mirai", quietly = TRUE) && mirai::status()$connections) {
```

Comment:
so this will automatically prefer mirai over future? Maybe there should be an option for this

Reply:
Only if the user also started a daemon with `mirai::daemon()`